python/xorbits/deploy/kubernetes/core.py

```diff
@@ -196,6 +199,42 @@ async def request_worker(
     ) -> str:
         raise NotImplementedError
 
+    async def request_workers(
+        self, worker_num: int, timeout: Optional[int] = None
+    ) -> List[str]:
+        if worker_num <= 0:
+            raise ValueError("Please specify a `worker_num` that is greater than zero")
+        if timeout and timeout < 0:
```

Comment:
Should this be timeout <= 0?

Reply:
I set timeout < 0 in order to distinguish between Timeout and ValueError test cases